game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/game/CarGameRoundProcessor.scala

```diff
@@ -15,6 +17,14 @@ class CarGameRoundProcessor extends GameRoundProcessor{
     val gamePlayers = carGameMap.getGamePlayers()
     val commandFactory = new CommandFactory
 
+    if (carGameMap.getCurrentRound >= Config.MAX_ROUNDS){
+      for (i <- gamePlayers.indices){
+        val player = gamePlayers(i).asInstanceOf[CarGamePlayer]
+        player.finish()
+      }
+      return true
+    }
+
```

Comment:
This part should probably be at the end of this method
If we leave it here, the bots will get to calculate a command on the very last round, but the command will be ignored because the MAX_ROUNDS check happens before this